user_docs/en/userGuide.t2t

```diff
@@ -1381,6 +1381,12 @@ However, in order for you to read the context (i.e. that you are in a list and t
 
 To toggle focus context presentation from anywhere, please assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+==== Interrupt speech while scrolling ====[BrailleSettingsInterruptSpeech]
+This setting determines if speech is interrupted when the braille display is scrolled.
+This option is enabled by default.
+
+Note that this only applies when scrolling the display backwards/forwards, previous/next line commands are not affected.

```

Comment:
This last line could be a little ambiguous.  Maybe a new sentence instead of a comma before "previous/next line commands are not affected."

Reply:
Fixed in 3410e8e